src/main/scala/org/allenai/deep_qa/pipeline/DropFirstColumnStep.scala

```diff
@@ -0,0 +1,60 @@
+package org.allenai.deep_qa.pipeline
+
+import com.mattg.pipeline.Step
+import com.mattg.util.FileUtil
+import com.mattg.util.JsonHelper
+import org.json4s._
+
+import org.allenai.deep_qa.data.DatasetReader
+
+import scala.sys.process.Process
+import scala.sys.process.ProcessLogger
+
+/**
+  * This Step is a SentenceProducer that reads a file with MCReadingComprehensionInstances
```

Comment:
You could make this comment a bit more general, because it's now just `DropFirstColumnStep`.  Leaving in the specific example for why this is useful is a good idea, though.